app/src/main/java/com/babylon/wallet/android/presentation/ui/modifier/DefaultCardShadow.kt

```diff
@@ -0,0 +1,22 @@
+package com.babylon.wallet.android.presentation.ui.modifier
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import com.babylon.wallet.android.designsystem.theme.RadixTheme
+
+fun Modifier.defaultCardShadow(
+    elevation: Dp,
+    shape: Shape? = null,
+    color: Color? = null
```

Comment:
Nice addition!

These 2 can also be non null and replaced with :
```
shape: Shape = RadixTheme.shapes.roundedRectMedium
color: Color = RadixTheme.colors.gray2
```
since they seem to never end up being null.

Reply:
this is Modifier extension, not a composable function so I can't use defaults from the theme